### PR TITLE
Allow hamburger icon in navbar to close sidebar on mobile

### PR DESCRIPTION
**Pull request by FyreByrd**
There was no good way to close the sidebar in mobile without choosing a navigation link in the sidebar, which is less than ideal UX.
I was able to fix this issue by moving the margin-top from the menu links within the sidebar to just the sidebar itself, so it would no longer be layered on top of the navbar and prevent clicks on the navbar from closing the sidebar.

### CODE REVIEW

**Comment by chrisvire** [comment]
@FyreByrd With UI changes, could you include before/after screenshots?

**Review by 7dev7urandom — APPROVED** [review]

**Comment by chrisvire** [comment]
@FyreByrd You don't need to wait for my approval...

**Pull request merged**
